File: 프로그래머스/unrated/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    sum_odd = 0
    sum_oven = 0
    
    for i, num in enumerate(num_list):      
        if i%2 == 0:
            sum_oven += num
        else:
            sum_odd += num           
    return sum_oven if sum_oven >= sum_odd else sum_odd

File: 프로그래머스/unrated/181887. 홀수 vs 짝수/홀수 vs 짝수.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int odd_sum = 0;
        int even_sum = 0;
        
        int i = 0;
        for(int num : num_list) {
            if(i%2 == 0)
                even_sum += num;
            
            if(i%2 != 0)
                odd_sum += num;
            
            i++;
        }
        return odd_sum > even_sum ? odd_sum : (odd_sum < even_sum ? even_sum : odd_sum);
    }
}